documentation/staging/content/userguide/managing-domains/domain-resource.md

```diff
@@ -103,21 +102,21 @@ DESCRIPTION:
-* `image`: The WebLogic container image; required when `domainHomeSourceType` is Image or FromModel; otherwise, defaults to container-registry.oracle.com/middleware/weblogic:12.2.1.4.
-* `imagePullPolicy`: The image pull policy for the WebLogic container image. Legal values are Always, Never, and IfNotPresent. Defaults to Always if image ends in :latest; IfNotPresent, otherwise.
+* `image`: The WebLogic container image; required when `domainHomeSourceType` is Image or FromModel (when not specifying an auxiliary image); otherwise, defaults to `container-registry.oracle.com/middleware/weblogic:12.2.1.4`. **WARNING**: The default image is unpatched and therefore, should always be specified to use an image with the latest patches installed (for example, `container-registry.oracle.com/middleware/weblogic_cpu:12.2.1.4-slim-jdk8-ol8`). For more information, see [Understand Oracle Container Registry images]({{< relref "/userguide/base-images/ocr-images#understand-oracle-container-registry-images" >}}).
+* `imagePullPolicy`: The image pull policy for the WebLogic container image. Legal values are Always, Never, and IfNotPresent. Defaults to Always if the image ends in `:latest`; IfNotPresent, otherwise. To ensure that all nodes have the same image, any images that are updated without changing the tag should be set to Always.
 * `imagePullSecrets`: A list of image pull Secrets for the WebLogic container image.
 * `domainHome`: The directory containing the WebLogic domain configuration inside the container. Defaults to /shared/domains/domains/<domainUID> if `domainHomeSourceType` is PersistentVolume. Defaults to /u01/oracle/user_projects/domains/ if `domainHomeSourceType` is Image. Defaults to /u01/domains/<domainUID> if `domainHomeSourceType` is FromModel.
-* `domainHomeSourceType`: Domain home file system source type: Legal values: Image, PersistentVolume, FromModel. Image indicates that the domain home file system is present in the container image specified by the `image` field. PersistentVolume indicates that the domain home file system is located on a persistent volume. FromModel indicates that the domain home file system will be created and managed by the operator based on a WDT domain model. If this field is specified, it overrides the value of `domainHomeInImage`. If both fields are unspecified, then `domainHomeSourceType` defaults to Image.
+* `domainHomeSourceType`: Domain home file system source type: Legal values: Image, PersistentVolume, FromModel. Image indicates that the domain home file system is present in the container image specified by the `image` field. PersistentVolume indicates that the domain home file system is located on a persistent volume. FromModel indicates that the domain home file system will be created and managed by the operator based on a WDT domain model.
```

Comment:
IMO, this should list the default value - but is it OK to say the default is 'Image' despite that it might not be depending on the deprecated `domainHomeInImage` field that we're trying to hide?

Reply:
It's probably time to remove those deprecated fields...

Reply:
We should remove all fields that have been deprecated as part of 4.0!